<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPicker</class>
 <widget class="QDialog" name="ColorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="2">
    <layout class="QHBoxLayout" name="Color_Alpha_Layout">
     <item>
      <widget class="QLabel" name="Color_Alpha_Label">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Color_Alpha_SpinBox">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QHBoxLayout" name="Color_Blue_Layout">
     <item>
      <widget class="QLabel" name="Color_Blue_Label">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Color_Blue_SpinBox">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="ColorPicker_Response">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="Color_Red_Layout_2">
     <item>
      <widget class="QLabel" name="Color_Red_Label">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>R:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Color_Red_SpinBox">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="Color_Alpha_Layout_2">
     <item>
      <widget class="QLineEdit" name="Color_Hex_LineEdit">
       <property name="inputMask">
        <string notr="true">\#HHHHHHHH</string>
       </property>
       <property name="text">
        <string>#000000</string>
       </property>
       <property name="cursorPosition">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="Color_Green_Layout_2">
     <item>
      <widget class="QLabel" name="Color_Green_Label">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>G:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="Color_Green_SpinBox">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ColorField">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.331579 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ColorPicker_Response</sender>
   <signal>accepted()</signal>
   <receiver>ColorPicker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorPicker_Response</sender>
   <signal>rejected()</signal>
   <receiver>ColorPicker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
